src/app/lib/config/toggles/README.md

```diff
@@ -0,0 +1,28 @@
+# Simorgh Application Toggles 
+
+| Toggle Name             | Description | Remote Toggle Enabled | 
+| ----------------------- | ----------- | --------------------- |
+| `ads`                   | Display Advertisements on Front Pages and Story (STY) Pages | ✅ | 
```

Comment:
With the remote toggle work still at PR stage, am I right in saying none of these toggles are really _enabled_ yet. Also I got the impression from yesterdays conversation that all toggles will be enabled for remote change via iSite but we will only surface certain ones to OPS in the runbook; have I got that right?

Reply:
Yeah good shout, I'll remove all but preroll (because this is mentioned in the runbook), and can add them back in as the toggles are live 👍